token/cli/src/main.rs

```diff
@@ -1916,6 +1919,104 @@ async fn command_sync_native(
     })
 }
 
+// Both enable_required_transfer_mesos and disable_required_transfer_mesos
+// Switches with enable_memos bool
+async fn command_required_transfer_memos(
+    token_account_address: Pubkey,
+    owner: Pubkey,
+    bulk_signers: BulkSigners,
+    enable_memos: bool,
+    config: &Config<'_>,
```

Comment:
nit: can you move this param to the top for consistency?

Reply:
resolved with 484279b